owrrm/src/main/java/com/facebook/openwifi/rrm/rca/RCAConfig.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.rrm.rca;
+
+import java.util.Map;
+
+/**
+ * Root cause analysis service configuration model.
+ */
+public class RCAConfig {
+	//
+	// NOTE:
+	// Currently assumes RCA is embedded in the RRM service and does NOT
+	// duplicate SDK-related fields.
+	//
+
+	/**
+	 * StationPinger parameters.
+	 */
+	public class StationPingerParams {
+		/**
+		 * How often to ping each station, in seconds (or 0 to disable)
+		 * ({@code STATIONPINGERPARAMS_PINGINTERVALSEC})
+		 */
+		// NOTE: cannot be shorter than Kafka "state" publish interval
+		public int pingIntervalSec = 0 /* TODO enable by default */;
+
+		/**
+		 * The number of pings to send to each station
+		 * ({@code STATIONPINGERPARAMS_PINGCOUNT})
+		 */
+		public int pingCount = 5;
+
+		/**
+		 * Number of executor threads for ping tasks
+		 * ({@code STATIONPINGERPARAMS_EXECUTORTHREADCOUNT})
+		 */
+		public int executorThreadCount = 3;
```

Comment:
How was the default 3 chosen?

Reply:
Randomly, suggestions welcome (now or later)